xikolo-ios/ViewControllers/Dashboard/CourseDatesTableViewController.swift

```diff
@@ -63,10 +72,25 @@ class CourseDatesTableViewController : UITableViewController {
 
     func updateAfterLoginStateChange() {
         self.navigationItem.rightBarButtonItem = UserProfileHelper.isLoggedIn() ? nil : self.loginButton
+        self.refresh()
+    }
 
+    func refresh() {
         self.tableView.reloadEmptyDataSet()
+        let deadline = 750.milliseconds.fromNow
+        let endSpinning = {
```

Comment:
Is there a reason for calling this one `endSpinning` when both other occurrences of the same code are `stopRefreshControl`?

Reply:
Yeah, this should also be `stopRefreshControl `.